### DISCUSSION TITLE
Add an exclamation mark (!) to the save button

### DISCUSSION THREAD

**Discussion by bouchtaoui-dev** [FlowiseAI/Flowise#746] [Ideas]
I think it's a nice feature to have an **!** on the save button as soon as there are unsaved changes.
I think this will improve the UX and it's easy to implement.

Hope you like the idea 🙂

**Comment by chungyau97** [comment]
Hi @bouchtaoui-dev,

There is a asterisk (*) when it required the flow to be save.
![image](https://github.com/FlowiseAI/Flowise/assets/33013947/ee73bc13-3a60-4041-831f-b54b63004dbd)

**Reply by bouchtaoui-dev** [reply]
Thnx for your answer, I didn't noticed it. Personally I think a badge on the save button would be better noticable. But that has no priority for now, however if you have some time, give it a second thought ;-)

**Reply by chungyau97** [reply]
Hi @bouchtaoui-dev,

Are you eager to embrace the challenge and contribute by submitting a pull request for this? 

Your involvement would be highly valued and deeply appreciated. 

cc @HenryHengZJ
